jetcd-core/src/main/java/io/etcd/jetcd/Cluster.java

```diff
@@ -42,9 +42,10 @@ public interface Cluster extends CloseableClient {
      * add a new member into the cluster.
      *
      * @param  peerAddrs the peer addresses of the new member
+     * @param  isLearner whether the member is raft learner
      * @return           the response
      */
-    CompletableFuture<MemberAddResponse> addMember(List<URI> peerAddrs);
+    CompletableFuture<MemberAddResponse> addMember(List<URI> peerAddrs, boolean isLearner);
```

Comment:
Is this mandatory or there is a default value ?

Reply:
it's better to be mandatory in my opinion

Reply:
do you have a link to the documentation on etcd ?

Reply:
> do you have a link to the documentation on etcd ?

https://github.com/etcd-io/etcd/blob/c2378be1b552663ac6c5278148aefbeb281b7228/api/etcdserverpb/rpc.proto#L879

Reply:
That's the proto definition, the documentation I was referring to is this one https://etcd.io/docs/v3.5/learning/design-learner/

Reply:
Since this is a breaking change, it would be nice to have a default method with the old behavior

Reply:
Sorry, I didn't find any etcd doc related with `isLearner`, but I added another `addMember` method for the old behavior